posix.mak

```diff
@@ -877,13 +877,13 @@ $G/changelog/next-version: ${DMD_DIR}/VERSION
 	@echo $(NEXT_VERSION) > $@
 
 changelog/prerelease.dd: $G/changelog/next-version $(LOOSE_CHANGELOG_FILES) | \
-							${STABLE_DMD} $(TOOLS_DIR) $(INSTALLER_DIR) $(DUB_DIR)
+							${STABLE_DMD} $(DMD_DIR) $(DRUNTIME_DIR) $(PHOBOS_DIR) $(TOOLS_DIR) $(INSTALLER_DIR) $(DUB_DIR)
```

Comment:
`druntime` and `phobos` weren't cloned by default for the changelog script anymore.